import_project.py

```diff
@@ -57,9 +57,9 @@ def _create_project(client, course_id, project_json, project_name=None):
 
 def _load_sandbox_images(client, course_id):
     # Update when we deploy 4.0 to include course-linked images
-    response = client.get('/api/sandbox_docker_images/')
+    response = client.get(f'/api/courses/{course_id}/sandbox_docker_images/')
```

Comment:
We should make an additional request here to `/api/sandbox_docker_images/` and then concatenate the two lists

Reply:
@james-perretta Tried to do this in 6f03972. I was able to import another project successfully. I'll admit that I didn't test it with a global image, but seeing as the `check_response` call didn't fail and the import worked, I think this should be good